src/components/common/TextInputComponent.jsx

```diff
@@ -2,26 +2,26 @@ import React from 'react';
 import { Input } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 
-const TextInputComponent = (props) => (
+const TextInputComponent = props => (
   <Input
     fluid
     size="large"
-    style={{width: "65%"}}
+    style={{ width: '65%' }}
     icon="pencil"
-    placeholder={props.placeholder}
+    placeholder={props.placeHolder}
```

Comment:
I hope you have verified that this change doesn't affect existing functionality/UI

Reply:
Yes I have verified the change does not affect existing functionality/UI.